### PR TITLE
run ica

### PR DESCRIPTION
**Pull request by kalenkovich**
can merge only after #3 and rebasing

### CODE REVIEW

**Comment by kalenkovich** [comment]
Rebased onto main. Ready for review, @levchenkoegor.

File: Snakefile [hunk 1]
```diff
@@ -20,6 +20,8 @@ def download_file_from_url(url, save_to):
 
 # Configuration constants
 L_FREQS = (None, 1)
+ICA_N_COMPONENTS = 0.999
```
Comment (levchenkoegor) [inline Snakefile:23]: Can you add a comment here and explain why a number of components for ICA is not an integer number?

**Comment by levchenkoegor** [comment]
Do we use somewhere ```ZipPath```?

It is imported on line 4:
```
from zipfile import ZipFile, Path as ZipPath
```

At my local brunch everything works well - can be merged.

File: Snakefile (hunk 1 above)
Comment (kalenkovich) [reply Snakefile:23]: Done.

**Pull request merged**